source/whats-new.txt

```diff
@@ -58,8 +98,8 @@ The v3.10.0 driver release includes the following new features:
   Root namespace declarations are still supported and expected to remain equivalent to
   their prior definitions. For example, ``bsoncxx::document`` is still equivalent to
   ``bsoncxx::v_noabi::document``, and ``mongocxx::client`` is still equivalent to
-  ``mongocxx::v_noabi::client``. Argument-dependent lookup and template instantiations
-  are expected to remain equivalent to their prior behavior.
+  ``mongocxx::v_noabi::client``. If this change breaks source or binary compatibility,
+  please submit a :ref:`bug report <cpp-issues-help>`.
```

Comment:
```suggestion
  ``mongocxx::v_noabi::client``.
```
As noted in an [earlier comment](https://github.com/mongodb/docs-cpp/pull/49#discussion_r1697466691), this recommendation is implicit. Recommend omitting it from the list.